src/graphql/root/query/on-chain-tx-fee-query.ts

```diff
@@ -0,0 +1,65 @@
+import { GT } from "@graphql/index"
+import * as Wallets from "@app/wallets"
+import * as Accounts from "@app/accounts"
+import { ApolloError } from "apollo-server-errors"
+import SatAmount from "@graphql/types/scalar/sat-amount"
+import WalletName from "@graphql/types/scalar/wallet-name"
+import OnChainTxFee from "@graphql/types/object/onchain-tx-fee"
+import OnChainAddress from "@graphql/types/scalar/on-chain-address"
+import TargetConfirmations from "@graphql/types/scalar/target-confirmations"
+
+const OnChainTxFeeQuery = GT.Field({
+  type: GT.NonNull(OnChainTxFee),
+  args: {
+    walletName: { type: WalletName },
+    address: { type: GT.NonNull(OnChainAddress) },
+    amount: { type: GT.NonNull(SatAmount) },
+    targetConfirmations: { type: TargetConfirmations, defaultValue: 3 },
+  },
+  resolve: async (_, args, { domainUser }) => {
+    const { walletName, address, amount, targetConfirmations } = args
```

Comment:
More of a question for my own understanding, when do we use `args` vs. `args.input` in these sorts of spots?

Reply:
I had the same question and what I understood (and validate doing some tests) was that makes easier to do changes to input params, i.e, you only need to change the query variables and not the query/mutation definition (for example in mobile app or in tests Ex: `test/integration/servers/graphql-main-server/mutations/ln-invoice-create.gql`). 

Please ask Samer for more details